Jamfile

```diff
@@ -463,10 +493,13 @@ feature.compose <encryption>off : <define>TORRENT_DISABLE_ENCRYPTION ;
 feature mutable-torrents : on off : composite propagated link-incompatible ;
 feature.compose <mutable-torrents>off : <define>TORRENT_DISABLE_MUTABLE_TORRENTS ;
 
-feature crypto : built-in openssl openssl-sha1 gcrypt : composite propagated ;
-feature.compose <crypto>openssl : <define>TORRENT_USE_OPENSSL <define>OPENSSL_NO_SSL2 ;
-feature.compose <crypto>openssl-sha1 : <define>TORRENT_USE_OPENSSL_SHA1 ;
-feature.compose <crypto>gcrypt : <define>TORRENT_USE_GCRYPT ;
+feature crypto : built-in libcrypto libgcrypt commoncrypto : composite propagated ;
```

Comment:
interesting. I guess I didn't have commonCrypto as a build feature. maybe it would make sense to just fold that into built-in. when building for mac/iOS I imagine it's fine (if not recommended) to just use that